apps/CMakeLists.txt

```diff
@@ -216,6 +216,7 @@ set(GDAL_DATA_FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/data/gdalinfo_output.schema.json
     ${CMAKE_CURRENT_SOURCE_DIR}/data/gdalmdiminfo_output.schema.json
     ${CMAKE_CURRENT_SOURCE_DIR}/data/ogrinfo_output.schema.json
+    ${CMAKE_CURRENT_SOURCE_DIR}/data/ogr_fields_override.schema.json
```

Comment:
I would rather move the schema to ogr/data/  because it is not specific to an application

Reply:
done